2.0/m_classban.cpp

```diff
@@ -0,0 +1,66 @@
+/*
+ * InspIRCd -- Internet Relay Chat Daemon
+ *
+ *   Copyright (C) 2016 Johanna Abrahamsson <johanna-a@mjao.org>
+ *  Backported from master (3.0) to 2.0 by genius3000, original at:
```

Comment:
You can just put a normal copyright line here, no need to say that its a backport.

Reply:
Fixed and noted for future, thanks.